src/LinuxTracing/ThreadStateManagerTest.cpp

```diff
@@ -19,6 +20,8 @@ using orbit_grpc_protos::ThreadStateSlice;
 
 TEST(ThreadStateManager, OneThread) {
```

Comment:
You should add a verification of `wakeup_reason` for every single slice in all these tests.

Reply:
This has only been addressed partially.

Reply:
It should be done now.